projects/dev/src/app/group/group-manage/pending-request/pending-request.component.html

```diff
@@ -98,29 +100,41 @@
       </ng-template>
       <ng-template #summaryTemplate let-table>
         <div style="text-align: left; display: flex; align-items: center;">
-          <p-tableHeaderCheckbox></p-tableHeaderCheckbox>
+          <p-tableHeaderCheckbox [disabled]="rejectLoading || acceptLoading"></p-tableHeaderCheckbox>
           <span
             style="
               margin-left: 2.1rem;
               color: #9b9b9b;
-              flex: 1;
               font-size: 1.2rem;
+              cursor: pointer;
             "
+            [ngClass]="{disabled: selection.length === 0 || rejectLoading || acceptLoading}"
+            (click)="onSelectAll($event)"
           >
             Select all
           </span>
+          <span style="flex: 1;"></span>
           <div class="req-tools">
-            <span class="accept" (click)="onClickAccept($event)">
+            <span class="accept" [ngClass]="{disabled: selection.length === 0 || rejectLoading || acceptLoading}" (click)="onClickAccept($event)">
               <span>Accept</span>
-              <i class="fa fa-check"></i>
+              <ng-container *ngIf="acceptLoading">
```

Comment:
The spinner doesn't grey out even it's under disabled state, because its styling has its own themeing so the outside state can't change it.

And for the disabled state text, I will make it more visible.

Reply:
fixed but I'd like you to review the disabled color if it does work for you.
And the progress spineer isn't affected by `disabled` state.
Added disabled cursor when the button is in a `disabled` state.

Reply:
The text is now more visible, looks ok.
However, the spinner is over the full text and icons, that does not look great :-/
![Screenshot 2020-05-15 at 15 52 27](https://user-images.githubusercontent.com/1053150/82057822-32e75800-96c4-11ea-82f4-9c18cf58fb83.png)
Is it possible to make it more visible?

Reply:
I also noticed that by testing and will update in the next commit.

Reply:
fixed.

Reply:
Why didn't you fix it before asking for review if you noticed it ? 😕

Reply:
I noticed that just before you send me the issue.
😁